ArduinoFrontend/src/environments/environment.prod.ts

```diff
@@ -5,5 +5,6 @@ export const environment = {
   production: true,
   API_URL: '../../',
   LOGIN_URL: `../../../eda/#/login?url=`,
-  DASHBOARD_URL: `../arduino/#/dashboard`
+  DASHBOARD_URL: `../arduino/#/dashboard`,
+  IMG_URL: 'http://localhost'
```

Comment:
I guess, this will create an issue in production.

Reply:
Yes correct, so in this case i have  remove only  value for that.